tests/conftest.py

```diff
@@ -72,16 +82,18 @@
     "settings": {
         "optimizer": {"enabled": True, "runs": 200},
         "outputSelection": {
-            "foo.sol": {"foo": ["abi", "bin", "bin-runtime", "devdoc", "userdoc"]},
-            "bar.sol": {"bar": ["abi", "bin", "bin-runtime", "devdoc", "userdoc"]},
+            "foo.sol": {"foo": ["abi", "bin", "bin-runtime", "devdoc", "userdoc", "srcmap"]},
```

Comment:
it looksl ike foo.sol is not compiling properly:
`E       AttributeError: ProjectManager has no attribute or contract named 'foo'.`

Reply:
resolving because I think the error might be with mylib as I mentioned in the other reply above?